website/docs/components/separator/partials/guidelines/guidelines.md

```diff
@@ -0,0 +1,44 @@
+## Usage
+
+### When to use
+
+- Use to visually separate different parts or sections on a page.
+- Use it to create a clear break between paragraphs, blocks, or sections.
+
+### When not to use
+
+- Don’t use to separate headings and body text. Instead, use the proper [Display token](https://github.com/hashicorp/design-system/issues/262) to emphasize hierarchy. 
+
+## Placement
+
+We recommend using meaningful spacing between content elements, allowing users to scan and understand information quickly. Use separators sparingly and purposefully to avoid cluttering the UI and distracting users from the main content.
```

Comment:
Suggesting a move of the spacing sentence to the proper section and adding some content from the "Usage" section. 

```suggestion
Use separators to distinguish between sections on a page visually. Use them sparingly and purposefully to avoid cluttering the UI and distracting users from the main content.
```